### PR TITLE
Update Jenkinsfile for OSDv4 migration. (#1008)

### PR DESCRIPTION
**Pull request by Hyperkid123**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,6 +70,96 @@ node {
...
+        // we have to wait 5-10s until akamai fast purge takes effect
+        sleep(10)
+      }
+    }
+  }
+
+  stage ("Run Akamai Production Smoke Tests") {
+    try {
+      if (RUN_SMOKE_TESTS) {
+        openShiftUtils.withNode {
```
Comment (quarckster) [inline Jenkinsfile:93]:
```groovy
openShiftUtils.withNode(
    image: "quay.io/cloudservices/iqe-tests:latest",
    namespace: "insights-dev-jenkins"
) {
```

File: Jenkinsfile [hunk 2]
```diff
@@ -66,6 +70,96 @@ node {
...
+          ]) {
+            // set some env variables for authentication in the iqe-core pod
+            withEnv([
+              "DYNACONF_IQE_VAULT_ROLE_ID=$DYNACONF_IQE_VAULT_ROLE_ID",
+              "DYNACONF_IQE_VAULT_SECRET_ID=$DYNACONF_IQE_VAULT_SECRET_ID",
+              "DYNACONF_IQE_VAULT_LOADER_ENABLED=true",
+              "ENV_FOR_DYNACONF=prod"
+            ]) {
+              // install akamai and 3scale plugins, run smoke tests
+              sh "iqe plugin install akamai 3scale"
```
Comment (quarckster) [inline Jenkinsfile:106]: this line is not needed

File: Jenkinsfile (hunk 1 above)
Comment (john-dupuy) [reply Jenkinsfile:93]: @quarckster are the `akamai` and `3scale` plugin images already included in the `iqe-tests:latest` image? 

Right now, this is just using the iqe-core image.

Comment (quarckster) [reply Jenkinsfile:93]: @john-dupuy yes, they are included https://url.corp.redhat.com/1d30796

Comment (Hyperkid123) [reply Jenkinsfile:93]: Ok Cool. We have the same setup in all other branches. I would like to merge this one first and update the image separately. Is that OK with you guys?

Comment (quarckster) [reply Jenkinsfile:93]: Yes, sounds good to me.

Comment (john-dupuy) [reply Jenkinsfile:93]: Works for me!

**Pull request merged**